datahub-web-react/src/Mocks.tsx

```diff
@@ -142,7 +142,8 @@ export const dataset1 = {
             displayName: 'HDFS',
             type: PlatformType.FileSystem,
             datasetNameDelimiter: '.',
-            logoUrl: '',
+            logoUrl:
+                'https://raw.githubusercontent.com/datahub-project/datahub/master/datahub-web-react/src/images/lookerlogo.png',
```

Comment:
is this change intended? cannot tell

Reply:
yeah - this is just for mocks to check if a logo url is there